### ISSUE TITLE
aezeed Cipher Seed Scheme does not work when using a passphase

### ISSUE THREAD

**Issue by openoms** [guggero/cryptography-toolkit#8]
I am experimenting with converting the onchain part of my LND wallet to an Electrum / Specter compatible format.
https://github.com/guggero/chantools `showrootkey` works great but I am having trouble with passphrases in https://guggero.github.io/cryptography-toolkit/#!/aezeed.
Typically I use 12 words Electrum seed as a passphrase, but putting that to the (offline) webapp just freezes it. Trying for further it freezes even with any passphrase.
Tried online also in Firefox and Chromium running on Debian Linux.

![image](https://user-images.githubusercontent.com/43343391/92498286-547bca80-f1f2-11ea-8123-ce19d3718d66.png)

**Comment by guggero** [comment]
Can confirm there's a problem in one of the libraries used. Apparently it breaks if the passphrase is longer or equal to 64 characters:
```
TypeError: p.push is not a function
    SHA256 https://guggero.github.io/cryptography-toolkit/libs/js/bitcoin.js:50228
    PBKDF2_HMAC_SHA256_OneIter https://guggero.github.io/cryptography-toolkit/libs/js/bitcoin.js:50255
    scrypt https://guggero.github.io/cryptography-toolkit/libs/js/bitcoin.js:50423
    generateSeed https://guggero.github.io/cryptography-toolkit/pages/aezeed/aezeed.js:65
```

Going to investigate how to fix this.

**Comment by guggero** [comment]
Was a problem in the `scrypt-js` library, thanks for reporting! Fixed in c8c5962.
